### PR TITLE
Add licence

### PR DESCRIPTION
**Pull request by annashipman**
All our repos should have licences.

### CODE REVIEW

File: LICENCE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (C) 2013 Crown Copyright (Government Digital Service)
```
Comment (cbaines) [inline LICENCE.txt]: I'd normally include a list or range of years, but just because I've seen this used elsewhere.

Comment (annashipman) [inline LICENCE.txt]: The range of years is used when significant changes are made in subsequent years. I don't know this repo well enough to know whether the core functionality dates from 2013 or whether there are subsequent changes that could qualify for copyright. What do you think?

More info here: http://www.copyrightservice.co.uk/copyright/p03_copyright_notices

Comment (cbaines) [inline LICENCE.txt]: Looking at that page, I'd say that this applies:
```
In the case of work which is continually updated, (for example a web site), the year of publication may be shown as a period from first publication until the most recent update, (i.e. 2000-2015)
```

But I don't really mind either way.

Comment (annashipman) [inline LICENCE.txt]: Looking back over the commit history it looks like there haven't been significant changes that might qualify for copyright in themselves, they are more keeping this app up to date with other apps, so I'd be inclined to leave it as 2013.

What does it mean that you've approved it? Does that mean I can merge it or are we waiting for someone else to?

**Comment by cbaines** [comment]
> What does it mean that you've approved it? Does that mean I can merge it or are we waiting for someone else to?

As this is part of GOV.UK, there is a process set out here [1], but it isn't publicly accessible. In this case, I've approved it as I'm happy for it to be merged, so I'd say that you are ok to go ahead and merge it if you want to.

1: https://gov-uk.atlassian.net/wiki/display/GOVUK/RFC+52+-+Pull+Request+merging+process

**Comment by annashipman** [comment]
👍 Thanks!

**Pull request merged**